lib/blockchain/chaindb.js

```diff
@@ -64,6 +64,10 @@ class ChainDB {
 
     this.cacheHash = new LRU(this.options.entryCache, null, BufferMap);
     this.cacheHeight = new LRU(this.options.entryCache);
+
+    // Keeps a copy of all NameStates in memory
```

Comment:
Good point thanks. I had originally set the cache size so large that it would always have every namestate for its entire auction cycle, but it used an extra 1.5 GB of RAM ;-)

Reply:
fixed 3525638b4de955b1400f915e42dd0104af623326